mvrx/src/test/kotlin/com/airbnb/mvrx/CoroutineStateStoreReplayTest.kt

```diff
@@ -23,18 +23,14 @@ class CoroutineStateStoreReplayTest {
 
     @Test
     fun replayTest() = runBlocking {
-        repeat(100) {
-            singleReplayTestIteration(N = 5000, subscribers = 10)
+        repeat(1000) {
+            // The store's backing SharedFlow has a buffer of 64 elements per subscriber. After that, it will drop
+            // the oldest messages.
+            singleReplayTestIteration(N = 64, subscribers = 10)
```

Comment:
potentially put 64 into an internal const to reuse the value with the state store implementation